vispy/app/application.py

```diff
@@ -77,6 +77,19 @@ def process_events(self):
         """
         return self._backend._vispy_process_events()
 
+    def sleep(self, duration_sec):
+        """
+        Sleep for the given duration. This is used to reduce
+        CPU stress when VisPy is run in interactive mode.
+        see inputhook.py for details
```

Comment:
@Eric89GXL: thanks for the docs! that definitely helps

also, which checker should I use ? [there seem to be a few comparisons on the web](http://stackoverflow.com/questions/1428872/pylint-pychecker-or-pyflakes). I've been using flake8 (in vim and sublime text). Should I change?